vendor-extra/JatsContentBundle/src/EventListener/BuildView/LinkListener.php

```diff
@@ -38,8 +38,12 @@ protected function canHandleElement(Element $element) : bool
         return string_is(
             $element->clarkNotation(),
             '{http://jats.nlm.nih.gov}aff',
+            '{http://jats.nlm.nih.gov}given-names',
             '{http://jats.nlm.nih.gov}kwd',
-            '{http://jats.nlm.nih.gov}subject'
+            '{http://jats.nlm.nih.gov}prefix',
+            '{http://jats.nlm.nih.gov}subject',
+            '{http://jats.nlm.nih.gov}suffix',
+            '{http://jats.nlm.nih.gov}surname'
```

Comment:
This is a bit annoying. We don't really want a Link, just text (which might have a `lang` attribute etc). Might need a Text pattern that is a `<span>` or not depending if there's attributes.